qucs/qucs/qucs.cpp

```diff
@@ -1589,6 +1605,7 @@ void QucsApp::slotFileSaveAll()
     // Hack! TODO: Maybe it's better to let slotFileChanged() know about Tab number?
     //   if saving was successful, turn off "saving needed" icon
     DocumentTab->setSaveIcon(!saveFile(Doc), No-1);
+    DocumentTab->setTabIcon(No-1,QPixmap(":/bitmaps/empty.xpm")); // know about Tab number?
```

Comment:
this does not seem correct, now a *Save all* clears the save icon also on the readonly file...